src/UIKit/UIApplication.cs

```diff
@@ -128,6 +128,26 @@ public static UIContentSizeCategory ToEnum (NSString key)
 			// Do not compare to Medium, it is the fallback
 			return UIContentSizeCategory.Medium;
 		}
+
+		public static NSString FromEnum (UIContentSizeCategory category)
```

Comment:
This also needs to be documented in our binding documentation, whenever I add new features to the generator, I make sure that the companion guides are updated.